### ISSUE TITLE
 Problem of warp_rnnt tensorflow in windows environment

### ISSUE THREAD

**Issue by jeonghwaYoo** [TensorSpeech/TensorFlowASR#73]
OS: Windows 10 , Anaconda
CUDA_toolkit: 10.1
Python: 3.7
Framework : tensorflow-gpu (2.3.0), pytorch(1.7.0)

Hello. Thank you for your projects
I have a problem with warprnnt
>No module named 'warprnnt_tensorflow'

When I run `./scripts/install_rnnt_loss.sh`, I got error message that `make` command could not be found. (The `cmake` command worked successfully. )

![image](https://user-images.githubusercontent.com/16534413/101877368-3ba7fa00-3bd1-11eb-907a-47461e7d9e2d.png)

So I used an alternative method.
`cmake --build . --target INSTALL --config Release`

However, another error message was displayed.
> Building NVCC (Device) object CMakeFiles/warprnnt.dir/src/Release/warprnnt_generated_rnnt_entrypoint.cu.obj
  nvcc fatal   : 32 bit compilation is only supported for Microsoft Visual Studio 2013 and earlier
  CMake Error at warprnnt_generated_rnnt_entrypoint.cu.obj.Release.cmake:220 (message):
    Error generating
    E:/Dev/Python/TEST/TensorFlowASR/externals/warp-transducer/build/CMakeFiles/warprnnt.dir/src/Release/warprnnt_gener
  ated_rnnt_entrypoint.cu.obj

>C:\Program Files (x86)\MSBuild\Microsoft.Cpp\v4.0\V140\Microsoft.CppCommon.targets(171,5): error MSB6006: "cmd.exe"exited.(code: 1). [E:\Dev\Python\TEST\TensorFlowASR\externals\warp-transducer\build\warprnnt.vcxproj]

How can I solve this problem? Or your project doesn't support windows?

**Comment by usimarit** [comment]
@jeonghwaYoo I'm not familiar with windows, so currently it's best if you use linux, or you can try modify cmake and compile with x64.

**Comment by jeonghwaYoo** [comment]
@usimarit  I already tried modify cmake and compile with x64, but another error was displayed.  So I solved it using another package (https://github.com/iamjanvijay/rnnt). Thank you.

**Comment by usimarit** [comment]
@jeonghwaYoo Can you review after using that package? Like how the loss would be, how fast is the training? I would really appreciate it.

**Comment by jeonghwaYoo** [comment]
@usimarit I think this packeage is not fully utilizes the CUDA warp mechanism, so I predict it will be slower than warp_rnnt. I will tell you more about it after training is over. Thank you.
